plugins/grafana-sankey-plugin/src/SankeyPanel.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { PanelProps } from '@grafana/data';
+import { SankeyOptions } from 'types';
+import Chart from 'react-google-charts';
+
+interface Props extends PanelProps<SankeyOptions> {}
+
+export const SankeyPanel: React.FC<Props> = ({ options, data, width, height }) => {
+  let result = [
+    ['From', 'To', 'Bytes'],
+    ['Source N/A', 'Destination N/A', 1],
+  ];
+  let sources = data.series
+    .map((series) => series.fields.find((field) => field.name === 'source'))
+    .map((field) => {
+      let record = field?.values as any;
+      return record?.buffer;
+    })[0];
+  if (sources !== undefined) {
+    let destinations = data.series
+      .map((series) => series.fields.find((field) => field.name === 'destination'))
+      .map((field) => {
+        let record = field?.values as any;
+        return record?.buffer;
+      })[0];
+    let destinationIPs = data.series
+      .map((series) => series.fields.find((field) => field.name === 'destinationIP'))
+      .map((field) => {
+        let record = field?.values as any;
+        return record?.buffer;
+      })[0];
+    let bytes = data.series
+      .map((series) => series.fields.find((field) => field.name === 'bytes'))
+      .map((field) => {
+        let record = field?.values as any;
+        return record?.buffer;
+      })[0];
+    let n = sources.length;
+    for (let i = 0; i < n; i++) {
+      let record = [];
+      let source = sources[i];
+      let destination = destinations[i];
+      record.push(source);
+      if (destination === '') {
+        record.push(destinationIPs[i]);
+      } else {
+        if (source === destination) {
+          destination = destination + ' ';
+        }
+        record.push(destination);
+      }
+      record.push(bytes[i]);
+      if (i === 0) {
+        result = [['From', 'To', 'Bytes']];
+      }
```

Comment:
nit: instead of re-initializing result you can consider adding the content at line 11 only if i === 0